perf/perf_UDT_probes.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2019 IBM
+# Author: Naveen kumar T <naveet89@in.ibm.com>
+
+import os
+import platform
+import shutil
+from avocado import Test
+from avocado import main
+from avocado.utils import distro, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class PerfProbe(Test):
+
+    def setUp(self):
+        '''
+        Install the basic packages to support perf and systemtap-sdt-devel.ppc64le
+        '''
+        # Check for basic utilities
+        smm = SoftwareManager()
+        detected_distro = distro.detect()
+        self.distro_name = detected_distro.name
+        deps = ['gcc', 'make']
+        if 'Ubuntu' in self.distro_name:
+            deps.extend(['linux-tools-common', 'linux-tools-%s' %
+                         platform.uname()[2]])
+        elif self.distro_name in ['rhel', 'SuSE', 'fedora', 'centos']:
```

Comment:
have we tested in fedora or centos as well ?

Reply:
i have tested in RHEL and SUSE. So, removing fedora and centos in code

Reply:
ok